roadie-rails.gemspec

```diff
@@ -20,9 +20,9 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_dependency "roadie", "~> 3.1"
-  spec.add_dependency "railties", ">= 3.0", "< 5.1"
+  spec.add_dependency "railties", ">= 3.0", "<= 5.1"
```

Comment:
You're correct, this should be `< 5.2`:

    2.4.0 :001 > Gem::Dependency.new('', '<= 5.1').match?('', '5.1.0')
     => true
    2.4.0 :002 > Gem::Dependency.new('', '<= 5.1').match?('', '5.1.1')
     => false